src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml

```diff
@@ -306,28 +213,51 @@
 
         <converters:BooleanToVisibilityNegationConverter x:Name="BooleanToVisibilityNegationConverter"/>
     </UserControl.Resources>
-
-    <Grid Margin="0,7,0,0" Background="Transparent">
+    <Grid>
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
-        <controls:EquationTextBox x:Name="EquationInputTextBox"
-                                  x:Uid="EquationInputButton"
-                                  Grid.Row="0"
-                                  Margin="0,0,3,0"
-                                  Style="{StaticResource KGF_EquationTextBoxStyle}"
-                                  Background="Transparent"
-                                  BorderThickness="0"
-                                  DataContext="{x:Bind ViewModel, Mode=OneWay}"
-                                  EquationButtonClicked="EquationButtonClicked"
-                                  EquationButtonContentIndex="{x:Bind ViewModel.FunctionLabelIndex, Mode=OneWay}"
-                                  EquationColor="{x:Bind ViewModel.LineColor, Mode=OneWay}"
-                                  Loaded="EquationInputTextBox_Loaded"/>
+        <Grid Padding="0,4,0,0">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="Auto"/>
+                <ColumnDefinition Width="*"/>
+            </Grid.ColumnDefinitions>
+            <Button x:Name="BackButton"
+                    x:Uid="equationAnalysisBack"
+                    MinWidth="44"
+                    MinHeight="44"
+                    VerticalAlignment="Stretch"
+                    Style="{StaticResource ButtonRevealStyle}"
+                    Background="{x:Bind ViewModel.LineColor}"
+                    Foreground="{ThemeResource SystemChromeWhiteColor}"
+                    BorderThickness="0"
+                    Click="BackButton_Click">
+                <StackPanel Margin="5,0"
+                            HorizontalAlignment="Center"
+                            VerticalAlignment="Center"
+                            Orientation="Horizontal">
+                    <FontIcon VerticalAlignment="Center"
+                              FontFamily="{ThemeResource SymbolThemeFontFamily}"
+                              FontSize="16"
+                              Glyph="&#xE72B;"/>
+                    <TextBlock Margin="12,0,0,0" Text="�"/>
+                    <TextBlock Margin="0,10,0,0"
+                               FontSize="9"
+                               Text="{x:Bind ViewModel.FunctionLabelIndex}"/>
+                </StackPanel>
+            </Button>
+            <controls:MathRichEditBox x:Name="EquationTextBox"
```

Comment:
Not this PR, but I noticed that very long equations get cut off since the RichEditBox is not editable, we cannot click into it to move the cursor to see the rest of the equation. Try a small window size with this equation:
`⁅𝑥+2+2+2+2+2+2+2+2+2+2+2+2+2+222222222222222222222222⁆`

Reply:
I've removed the ` scrollviewer.xxx = Disabled` properties from the style!